cyral/resource_cyral_sidecar_listener.go

```diff
@@ -0,0 +1,363 @@
+package cyral
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/cyralinc/terraform-provider-cyral/client"
+)
+
+// create a constant block for schema keys
+
+const (
+	SidecarIdKey        = "sidecar_id"
+	ListenerIdKey       = "listener_id"
+	RepoTypesKey        = "repo_types"
+	NetworkAddressKey   = "network_address"
+	PortKey             = "port"
+	HostKey             = "host"
+	FileKey             = "file"
```

Comment:
I believe the `UnixListener` type got remove from the `ListenerConfig` design, right? If thats case, we may not need this `file` argument anymore.

Reply:
True! Good catch, will remove.